optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/ClassAndPlanningIdComparator.java

```diff
@@ -14,41 +14,32 @@
  * limitations under the License.
  */
 
-package org.optaplanner.core.impl.domain.lookup;
+package org.optaplanner.core.impl.score.director;
 
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.HashMap;
 import java.util.Map;
 
 import org.optaplanner.core.api.domain.common.DomainAccessType;
-import org.optaplanner.core.api.domain.lookup.PlanningId;
 import org.optaplanner.core.config.util.ConfigUtils;
 import org.optaplanner.core.impl.domain.common.accessor.MemberAccessor;
 
-public class ClassAndPlanningIdComparator implements Comparator<Object> {
+final class ClassAndPlanningIdComparator implements Comparator<Object> {
```

Comment:
I'd still make it "final public", in case anyone is using it. It's in impl, so they aren't allowed to, but no gain in breaking them if they do.

```suggestion
final public class ClassAndPlanningIdComparator implements Comparator<Object> {
```